distributions/aws/examples/storage-efs/README.md

```diff
@@ -0,0 +1,153 @@
+
+1. Download the IAM policy document from GitHub as follows - 
+
+```
+curl -o iam-policy-example.json https://raw.githubusercontent.com/kubernetes-sigs/aws-efs-csi-driver/v1.3.4/docs/iam-policy-example.json
+```
+
+2. Create the policy - 
+```
+aws iam create-policy \
+    --policy-name AmazonEKS_EFS_CSI_Driver_Policy \
+    --policy-document file://iam-policy-example.json
+```
+
+3. Create an IAM role and attach the IAM policy to it. Annotate the Kubernetes service account with the IAM role ARN and the IAM role with the Kubernetes service account name. You can create the role using eksctl as follows - 
+
+```
+eksctl create iamserviceaccount \
+    --name efs-csi-controller-sa \
+    --namespace kube-system \
+    --cluster $CLUSTER_NAME \
+    --attach-policy-arn arn:aws:iam::$AWS_ACCOUNT_ID:policy/AmazonEKS_EFS_CSI_Driver_Policy \
+    --approve \
+    --override-existing-serviceaccounts \
+    --region $CLUSTER_REGION
+```
+
+4. You can verify by describing the specified service account to check if it has been correctly annotated - 
+```
+kubectl describe -n kube-system serviceaccount efs-csi-controller-sa
+```
+
+## 4.0 Create an Instance of the EFS Filesystem
+Please refer to the official [AWS EFS CSI Document](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html#efs-create-filesystem) for detailed instructions on creating an EFS filesystem. 
+
+
+## 5.0 Using EFS Storage in Kubeflow
+
+## 5.1 Provisioning Options
+### Option 1 - Static Provisioning
+[Using this sample from official AWS Docs](https://github.com/kubernetes-sigs/aws-efs-csi-driver/tree/master/examples/kubernetes/multiple_pods) we have provided the required spec files in the sample subdirectory but you can create the PVC another way. 
+
+1. Use the `$file_system_id` you recorded in section 4 above or use the AWS Console to get the filesystem id of the EFS volume you want to use. You could also use the following command to list all the volumes available in your region. Either way, make sure that `file_system_id` is set. 
+```
+aws efs describe-file-systems --query "FileSystems[*].FileSystemId" --output text --region $CLUSTER_REGION
+```
+
+2. Now edit the last line of the sample/pv.yaml file to specify the `volumeHandle` field to point to your EFS filesystem.
+```
+yq e '.spec.csi.volumeHandle = env(file_system_id)' -i sample/pv.yaml
+```
+
+3. The `PersistentVolume` and `StorageClass` are cluster scoped resources but the `PersistentVolumeClaim` needs to be in the namespace you will be accessing it from. Be sure to replace the `kubeflow-user-example-com` namespace specified in the `sample/pvc.yaml` file with the namespace for the kubeflow user. 
+
+4. Now create the required persistentvolume, persistentvolumeclaim and storageclass resources as -
+```
+kubectl apply -f storage-efs/sample/pv.yaml
+kubectl apply -f storage-efs/sample/pvc.yaml
+kubectl apply -f storage-efs/sample/sc.yaml
+```
+
+## 5.2 Check your Setup
```

Comment:
Sorry I missed that but you don't need to see dashboard to verify the volumes are up.

I think command on `get pvc` should tell you the status of `pvc` whether it has been `bounded` or not

Reply:
Either option works and we are anyway logging onto the dashboard to create a notebook, any reason to prefer one over the other ?

Reply:
I guess its fine to check the dashboard too..but like how you're verifying the csi driver, you can verify the pvc is bounded or not. This maintains the consistency in your instructions.

Reply:
Adding both options.